docker/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:latest
+
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install -y nodejs 
+
+RUN mkdir -p /usr/src/app
+
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app
+
+RUN npm install
+
+COPY . /usr/src/app
+
+EXPOSE 7500
```

Comment:
If you've exposed 7500, you should also set `PORT` to `7500` to make sure that the app listens on the exposed port.